src/reference/config/bindings/binding-openapi-asyncapi.md

```diff
@@ -0,0 +1,161 @@
+---
+shortTitle: openapi-asyncapi
+description: Zilla runtime openapi-asyncapi binding
+category:
+  - Binding
+tag:
+  - Proxy
+---
+
+# openapi-asyncapi Binding
+
+Zilla runtime openapi-asyncapi binding.
+
+```yaml {2}
+openapi_asyncapi_proxy:
+  type: openapi-asyncapi
+  kind: proxy
+  options:
+    specs:
+      openapi:
+        openapi-id: spec/openapi.yaml
+      asyncapi:
+        asyncapi-id: spec/asyncapi.yaml
+  routes:
+    - when:
+        - api-id: openapi-id
+      exit: asyncapi_client
+      with:
+        api-id: asyncapi-id
+```
+
+## Summary
+
+The `proxy` kind `openapi-asyncapi` binding adapts `opanapi` request-response streams to `asyncapi` streams.
+
+## Configuration
+
+:::: note Properties
+
+- [kind\*](#kind)
+- [options](#options)
+  - [options.specs](#options-specs)
+    - [specs.openapi](#specs-openapi)
+    - [specs.asyncapi](#specs-asyncapi)
+- [routes](#routes)
+  - [routes\[\].when](#routes-when)
+    - [when\[\].api-id](#when-api-id)
+    - [when\[\].operation-id](#when-operation-id)
+  - [routes\[\].exit\*](#routes-exit)
+  - [routes\[\].with](#routes-with)
+    - [with.api-id](#with-api-id)
+    - [with.operation-id](#with-operation-id)
+
+::: right
+\* required
+:::
+
+::::
+
+### kind\*
+
+> `enum` [ "proxy" ]
+
+Behave as an `openapi-asyncapi` `proxy`.
+
+```yaml
+kind: proxy
+```
+
+### options
+
+> `object`
+
+`openapi-asyncapi` - specific options.
+
+```yaml
+options:
+  specs:
+    openapi:
+      openapi-id: spec/openapi.yaml
+    asyncapi:
+      asyncapi-id: spec/asyncapi.yaml
+```
+
+#### options.spec
```

Comment:
The descriptions say filename and unique id, does that mean the filename is also the id? I don't understand the mention of a unique id.